pytest/tests/sanity/restart.py

```diff
@@ -17,7 +17,7 @@
 
 nodes = start_cluster(
     2, 0, 2, None,
-    [["epoch_length", 10], ["block_producer_kickout_threshold", 80]], {})
+    [["epoch_length", 10], ["block_producer_kickout_threshold", 0]], {})
```

Comment:
Can you add a short comment here describing why this is needed?

Reply:
:+1: Added the comment